example_test.go

```diff
@@ -388,6 +388,60 @@ func ExampleObjects() {
 	// {"level":"debug","msg":"opening connections","addrs":[{"ip":"123.45.67.89","port":4040},{"ip":"127.0.0.1","port":4041},{"ip":"192.168.0.1","port":4042}]}
 }
 
+func ExampleDictObject() {
+	logger := zap.NewExample()
+	defer logger.Sync()
+
+	// Use DictObject to create zapcore.ObjectMarshaler implementations from Field arrays,
+	// then use the Object and Objects field constructors to turn them back into a Field.
+
+	logger.Debug("worker received job",
+		zap.Object("w1",
+			zap.DictObject(
+				zap.Int("id", 402000),
+				zap.String("description", "compress image data"),
+				zap.Int("priority", 3),
+			),
+		))
+
+	d1, err := time.ParseDuration("68ms")
+	if(err != nil) {
+		panic(err)
+	}
+	d2, err := time.ParseDuration("79ms")
+	if(err != nil) {
+		panic(err)
+	}
+	d3, err := time.ParseDuration("57ms")
+	if(err != nil) {
+		panic(err)
+	}
```

Comment:
can we just use time.Duration values directly here?

```suggestion
	d1 := 68 * time.Millisecond
	d2 := 79 * time.Millisecond
	d3 := 57 * time.Millisecond
```

Reply:
yep, this looks much better